src/main/java/com/spotinst/sdkjava/model/api/azure/statefulNode/ApiStatefulNodeComputeConfiguration.java

```diff
@@ -0,0 +1,126 @@
+package com.spotinst.sdkjava.model.api.azure.statefulNode;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Created by Bansi Chapla on 23/11/2021.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiStatefulNodeComputeConfiguration implements IPartialUpdateEntity {
+
+    @JsonIgnore
+    private Set<String>                         isSet;
+    private String                              os;
+    private ApiStatefulNodeVmSizesConfiguration vmSizes;
+    private List<String>                        zones;
+    private String                              preferredZone;
```

Comment:
preferredZone should be preferredZones and it should be an array of strings.

Reply:
document error